data/processor.py

```diff
@@ -469,11 +469,11 @@ def _opendata_to_section_meeting(data, term_year):
                 date['weekdays'].replace('U', 'Su'))
 
     # TODO(david): Actually use the term begin/end dates when we get nulls
-    date_format = '%m/%d'
-    start_date = datetime.strptime(date['start_date'], date_format).replace(
-            year=term_year) if date['start_date'] else None
-    end_date = datetime.strptime(date['end_date'], date_format).replace(
-            year=term_year) if date['end_date'] else None
+    date_format = '%m/%d/%Y'
+    start_date = datetime.strptime(date['start_date'] + '/' + str(term_year),
+        date_format).replace(year=term_year) if date['start_date'] else None
```

Comment:
1) Ignored!
2) Good call, we shouldn't need replace anymore
3) Good call. Wasn't really a fan of the formatting either